repos/debian/rpi4/config/multistrap.conf

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright (c) 2021 Sartura Ltd.
+#
+# Example configuration
+#
+[General]
+cleanup=true
+noauth=true
+setupscript=/usr/share/multistrap/chroot.sh
+bootstrap=Debian Foreign
+aptsources=Debian
+
+[Debian]
+packages=firmware-brcm80211 ca-certificates curl gnupg lsb-release iw
+# The following packages are marked as important by aptitude and are not included by default
+packages=adduser apt-utils cpio cron debconf-i18n dmidecode debian-archive-keyring dmidecode
+packages=fdisk gpgv ifupdown init iproute2 iputils-ping isc-dhcp-client isc-dhcp-common kmod
+packages=less libreadline8 logrotate nano netbase nftables procps readline-common rsyslog
+packages=sensible-utils systemd systemd-sysv tasksel-data udev vim-common vim-tiny whiptail
+source=http://deb.debian.org/debian/
+keyring=debian-archive-keyring
+suite=stable
+omitdebsrc=false
+components=main contrib non-free
+
+[Foreign]
+packages=docker-ce docker-ce-cli containerd.io
```

Comment:
We shouldn't install this by default. Let's leave these commented out, but we should ensure this works.